sdk/trace/src/main/java/io/opentelemetry/sdk/trace/ExtendedSdkTracer.java

```diff
@@ -12,16 +12,12 @@
 
 /** {@link ExtendedSdkTracer} is SDK implementation of {@link ExtendedTracer}. */
 final class ExtendedSdkTracer extends SdkTracer implements ExtendedTracer {
-  // TODO: add dedicated API for updating scope config.
-  @SuppressWarnings("FieldCanBeFinal") // For now, allow updating reflectively.
-  private boolean tracerEnabled;
```

Comment:
Needed to make sure the updates are visible to `ExtendedTracer#isEnabled()`. Updated `SdkTracer.tracerEnabled` to be protected, and removed this field, such that `ExtendedSdkTracer#isEnabled()` resolves using the value from `SdkTracer.tracerEnabled`.